CHANGELOG.adoc

```diff
@@ -71,6 +71,11 @@ This incorporates the following executed proposals:
 
 == DFX
 
+=== fix: Initialize replica with bitcoin regtest flag
+
+When the bitcoin feature is enabled, dfx was launching the replica with the "bitcoin_testnet" feature.
+The correct feature to use is "bitcoin_regtest".
+
```

Comment:
This needs to go in the "unreleased" section above

Reply:
Done.